mirrormanager2/app.py

```diff
@@ -373,14 +375,22 @@ def host_new(site_id):
 
         try:
             SESSION.flush()
+            status = 'Host added'
```

Comment:
@ralphbean @pypingou yes, updated to "host.added". Also, updated all the edits as discussed. Please review them. Thankyou :)